### PR TITLE
TSE-1265 Branch and update examples

### PR DESCRIPTION
**Pull request by elkl-trmb**
--> Fixes most build errors

### CODE REVIEW

File: Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml [hunk 1]
```diff
@@ -24,8 +24,8 @@
 		<SourcePathVariable Id="SourceModelPlugins" Value="%SourceBin%\plugins\Tekla\Model\%ExtensionName%"/>
 		<SourcePathVariable Id="SourceDrawingApps" Value="%SourceBin%\applications\Tekla\Drawings\%ExtensionName%"/>
 		<SourcePathVariable Id="SourceModelApps" Value="%SourceBin%\applications\Tekla\Model\%ExtensionName%"/>
-		<SourcePathVariable Id="SourceCommmon" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\common"/>
-		<SourcePathVariable Id="SourceUsaRoot" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\USA"/>
+		<!-- <SourcePathVariable Id="SourceCommmon" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\common"/> -->
+		<!-- <SourcePathVariable Id="SourceUsaRoot" Value="%TEPDEFINITIONFILEFOLDER%\ProgramData\Environments\USA"/> -->
```
Comment (CharlesPool) [inline Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml:28]: Why are these now turned off?

File: Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml [hunk 2]
```diff
@@ -46,8 +46,8 @@
 		<!-- <File Id="RibbonImages" Source="%SourceImages%\*.*" Target="%ExtensionsDir%\Images" Recursive="yes" /> -->
 		<File Id="PluginFiles" Source="%SourceModelPlugins%\*.*" Target="%ModelPluginsDir%" Recursive="yes" />
 		<File Id="TranslationsFiles" Source="%SourceMessages%\*.*" Target="%ModelPluginsDir%" Recursive="yes" />
-		<File Id="GlobalCommon" Source="%SourceCommmon%\*.*" Target="%TargetCommonDir%" Recursive="yes" />
-		<File Id="USARootFiles" Source="%SourceUsaRoot%\*.*" Target="%TargetUsaRoot%" Recursive="yes" />
+		<!-- <File Id="GlobalCommon" Source="%SourceCommmon%\*.*" Target="%TargetCommonDir%" Recursive="yes" /> -->
+		<!-- <File Id="USARootFiles" Source="%SourceUsaRoot%\*.*" Target="%TargetUsaRoot%" Recursive="yes" /> -->
```
Comment (CharlesPool) [inline Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml:50]: Why are these now turned off?

File: Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs [hunk 3]
```diff
@@ -103,7 +103,7 @@ private void DetachHandlers()
         private void Polygon_OnPreviewRequested(object sender, ToleratedObjectEventArgs eventArgs)
         {
             Graphics.Clear();
-            var currentAppliedValues = GetAppliedAttributes(Component);
+            //var currentAppliedValues = GetAppliedAttributes(Component);     //this method seems to have been removed
```
Comment (CharlesPool) [inline Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs:107]: Then add call to get values from Component via extension method direct so data is refreshed please.

File: Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs [hunk 4]
```diff
@@ -201,7 +201,7 @@ private void Guideline_OnPickingUndone(object sender, EventArgs eventArgs)
         private void Guideline_OnPreviewRequested(object sender, ToleratedObjectEventArgs eventArgs)
         {
             Graphics.Clear();
-            var currentAppliedValues = GetAppliedAttributes(Component);
+            //var currentAppliedValues = GetAppliedAttributes(Component);
```
Comment (CharlesPool) [inline Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs:205]: Add direct call to Component to fetch dialog data

**Review by CharlesPool — CHANGES_REQUESTED** [review]

File: Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml (hunk 2 above)
Comment (elkl-trmb) [reply Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml:50]: the folders are empty or non existent in the repo

File: Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml (hunk 1 above)
Comment (elkl-trmb) [reply Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml:28]: the folders are empty or non existent in the repo

File: Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs (hunk 3 above)
Comment (elkl-trmb) [reply Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs:107]: the variable is not used elsewhere

File: Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs (hunk 4 above)
Comment (elkl-trmb) [reply Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs:205]: the variable is not used elsewhere

**Review by CharlesPool — APPROVED** [review]

**Pull request merged**